test/Test/Oscoin/API.hs

```diff
@@ -24,8 +24,8 @@ import           Test.Tasty.HUnit.Extended
 import           Test.Tasty.QuickCheck
 
 
-tests :: forall c. Dict (IsCrypto c) -> [TestTree]
-tests Dict =
+tests :: forall c. Dict (IsCrypto c) -> TestTree
+tests Dict = testGroup "Oscoin.API"
```

Comment:
Wait .. shouldn't this always be _literally_ the module's name?

Reply:
I’d be ok with that, too. But I omitted the `Test` prefix in all my changes and this is also what is documented in [“Test organization”](https://github.com/oscoin/oscoin#test-organization) in the readme.

Reply:
Ok, guess I missed that. I think mechanically the module name is preferable, as we can eventually just have it generated.

Reply:
I’ll switch to that